From 1a04a9a59830705dd65116354e862fd6a658722e Mon Sep 17 00:00:00 2001
From: Adam Ford <aford173@gmail.com>
Date: Wed, 2 Jun 2021 14:34:08 -0500
Subject: [PATCH 6/6] configs: renesas: beacon: Enable GPIO Hog

The GPIO Hog is used to hold the USB out of reset.  Enable
the hog driver so it can do this.

Signed-off-by: Adam Ford <aford173@gmail.com>
---
 configs/r8a774a1_beacon_defconfig | 1 +
 configs/r8a774b1_beacon_defconfig | 1 +
 configs/r8a774e1_beacon_defconfig | 1 +
 3 files changed, 3 insertions(+)

diff --git a/configs/r8a774a1_beacon_defconfig b/configs/r8a774a1_beacon_defconfig
index 11b63299a8..dc7018b7b2 100644
--- a/configs/r8a774a1_beacon_defconfig
+++ b/configs/r8a774a1_beacon_defconfig
@@ -41,6 +41,7 @@ CONFIG_CLK=y
 CONFIG_CLK_CCF=y
 CONFIG_CLK_RENESAS=y
 CONFIG_CLK_VERSACLOCK=y
+CONFIG_GPIO_HOG=y
 CONFIG_RCAR_GPIO=y
 CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
diff --git a/configs/r8a774b1_beacon_defconfig b/configs/r8a774b1_beacon_defconfig
index 8170383e6c..a3b36fc946 100644
--- a/configs/r8a774b1_beacon_defconfig
+++ b/configs/r8a774b1_beacon_defconfig
@@ -41,6 +41,7 @@ CONFIG_CLK=y
 CONFIG_CLK_CCF=y
 CONFIG_CLK_RENESAS=y
 CONFIG_CLK_VERSACLOCK=y
+CONFIG_GPIO_HOG=y
 CONFIG_RCAR_GPIO=y
 CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
diff --git a/configs/r8a774e1_beacon_defconfig b/configs/r8a774e1_beacon_defconfig
index 9d9e3ba667..e484210783 100644
--- a/configs/r8a774e1_beacon_defconfig
+++ b/configs/r8a774e1_beacon_defconfig
@@ -41,6 +41,7 @@ CONFIG_CLK=y
 CONFIG_CLK_CCF=y
 CONFIG_CLK_RENESAS=y
 CONFIG_CLK_VERSACLOCK=y
+CONFIG_GPIO_HOG=y
 CONFIG_RCAR_GPIO=y
 CONFIG_DM_PCA953X=y
 CONFIG_DM_I2C=y
-- 
2.25.1

